packages/cms/pages/resources/resource/resource.tsx

```diff
@@ -189,10 +189,26 @@ export const FilterList: React.FunctionComponent = () => {
 export const Table: React.FunctionComponent = () => {
   const [pageSize, setPageSize] = useState(10)
   const [pageIndex, setPageIndex] = useState(0)
+
   const [selectedItems, setSelectedItems] = useState<any[]>([])
   const [sortField, setSortField] = useState('firstName')
   const [sortDirection, setSortDirection] = useState<'asc' | 'desc'>('asc')
-  const { resource, applyFilter } = useResourceStore()
+  const { resource, applyFilter, fetchTableData } = useResourceStore()
+  const [items, setItems] = useState([])
+
+  useEffect(() => {
+    const getData = async () => {
+      const [{ data }, error] = await fetchTableData(
+        resource?.slugPlural.toLowerCase()!,
```

Comment:
If the `fetchTableData` function is coming from the store, then we probably don't need to pass the resource slug from here. Maybe we can just get it from the store and use it in there 🤔 . 

Also, we can just use `resource?.slug`.

So in the store we get the resource using:

```
const { resource } = get()

```